src/backbone/backbone.c

```diff
@@ -94,9 +94,11 @@ static int backbone_cb_id(struct backbone_id *id)
 
 static bool backbone_cb_update(void)
 {
-	struct backbone_id new = { 0 }, *id;
+	/* struct backbone_id = {{ 0 }}, *id; */  /* correctly zero-initialized */
```

Comment:
Just drop this line instead of commenting it out, we have git for file history :)